src/clients/composite/aws_vpc_client.py

```diff
@@ -76,15 +75,64 @@ def _is_flow_log_misconfigured(self, flow_log: FlowLog) -> bool:
             or not flow_log.deliver_log_role_arn.endswith(f":role/{self.config.logs_vpc_log_group_delivery_role()}")
         )
 
-    def enforcement_actions(self, vpcs: Sequence[Vpc], with_subscription_filter: bool) -> Sequence[ComplianceAction]:
+    def enforcement_flow_log_actions(
+        self, vpcs: Sequence[Vpc], with_subscription_filter: bool
+    ) -> Sequence[ComplianceAction]:
         if not vpcs:
             return list()
-        log_group_actions = self._vpc_log_group_enforcement_actions(with_subscription_filter)
+        log_group_config = self.config.logs_vpc_flow_log_group_config()
+        log_group_actions = self.log_group.log_group_enforcement_actions(
+            log_group_config=log_group_config, with_subscription_filter=with_subscription_filter
+        )
         delivery_role_actions = self._delivery_role_enforcement_actions()
-        vpc_actions = [action for vpc in vpcs for action in self._vpc_enforcement_actions(vpc)]
+        vpc_actions = [action for vpc in vpcs for action in self._vpc_flow_enforcement_actions(vpc)]
         return list(chain(log_group_actions, delivery_role_actions, vpc_actions))
 
-    def _vpc_enforcement_actions(self, vpc: Vpc) -> Sequence[ComplianceAction]:
+    def enforcement_dns_log_actions(
+        self, vpcs: Sequence[Vpc], with_subscription_filter: bool
+    ) -> Sequence[ComplianceAction]:
+        if not vpcs:
+            return []
+
+        log_group_config = self.config.logs_vpc_dns_log_group_config()
+        log_group_actions = self.log_group.log_group_enforcement_actions(
+            log_group_config=log_group_config, with_subscription_filter=with_subscription_filter
+        )
+
+        resolver_config = self._resolver_query_log_config_enforcement_actions(
+            log_group_config=log_group_config, vpcs=vpcs
+        )
+
+        # vpc_actions = [action for vpc in vpcs for action in self.
+        # _vpc_resolver_query_log_config_associations_enforcement_actions(vpc=vpc,
+        # log_group_config=log_group_config)]
+
+        # def _vpc_resolver_query_log_config_associations_enforcement_actions(self, vpc: Vpc,
+        # log_group_config: LogGroupConfig) -> Sequence[ComplianceAction]:
+        #     return [CreateResolverQueryLogConfigAssociation(log=self.logs, resolver=self.resolver,
```

Comment:
we have some commented out code here, but plan to address this in the next commit/pr